### ISSUE TITLE
Jar files needed for running the docker image

### ISSUE THREAD

**Issue by mostafaghadimi** [JosemyD/docker-spark-mongo#2]
Hi @JosemyDuarte, first of all I should be thankful for your nice image of spark to mongodb connector. I need the jarfiles which is specified by environment variables named JARS and SPARK_DRIVER_EXTRA_CLASSPATH. How can I download them and put in the same directory?
What I want is to dockerize everything (Spark, spark-mongodb-connector, MongoDB)

The error I faced with is the following:

```
ERROR: for spark-mongo  Cannot start service spark-mongo: OCI runtime create failed: container_linux.go:345: starting container process caused "eStarting mongo ... done

ERROR: for spark-mongo  Cannot start service spark-mongo: OCI runtime create failed: container_linux.go:345: starting container process caused "exec: \"pyspark --jars  --driver-class-path \": executable file not found in $PATH": unknown
ERROR: Encountered errors while bringing up the project.
```

and this is my docker-compose file:

```
version: '3'

services: 
  mongo:
    image: mongo
    container_name: mongo
    ports: 
      - 27017:27017

  spark-mongo:
    image: josemyd/docker-spark-mongo
    container_name: spark-mongo
    command: 
      - pyspark --jars ${JARS} --driver-class-path ${SPARK_DRIVER_EXTRA_CLASSPATH}

```

**Comment by JosemyDuarte** [comment]
Hi @mostafaghadimi! I just updated the repo with an easier way to get started. I added a docker-compose, upgrade to Spark 2.4.3 and updated the README.md. Take a look at the latest changes. 

I'm not sure about the source of your problem, I think it could be something related to how you were passing arguments to the containers. Anyway, it should work with the latest changes. 

The JARS that you were asking me are inside the container, you shouldn't need to have them in your local machine.
